### PR TITLE
Add service restart policy, doc, defaults

### PR DESCRIPTION
**Pull request by rcarmo**
I found it rather strange that my deployment failed upon rebooting the host... This fixes it, of course.

### CODE REVIEW

**Comment by c-goes** [comment]
I would prefer `unless-stopped`, still it's a step in the right direction.

**Comment by sapkra** [comment]
Duplicate of https://github.com/jitsi/docker-jitsi-meet/pull/377

**Comment by rcarmo** [comment]
Here you go, all done. Kindly merge if you agree.

File: env.example [hunk 1]
```diff
@@ -317,3 +317,7 @@ JIBRI_LOGS_DIR=/config/logs
 # Redirect HTTP traffic to HTTPS
 # Necessary for Let's Encrypt, relies on standard HTTPS port (443)
 #ENABLE_HTTP_REDIRECT=1
+
+# Container restart policy
+# Defaults to always-stopped
```
Comment (albe) [inline env.example:322]:
```suggestion
# Defaults to unless-stopped
```

**Pull request merged**